crates/trie/trie/src/state.rs

```diff
@@ -50,33 +50,6 @@ impl HashedPostState {
         Self { accounts, storages }
     }
 
-    /// Initialize [`HashedPostState`] from cached state.
-    /// Hashes all changed accounts and storage entries that are currently stored in cache.
-    pub fn from_cache_state<'a>(
```

Comment:
this function is never used in the codebase and is only part of a public API. Removed in order to avoid having to maintain unused code.

Reply:
I think we should keep this in to keep diff with upstream minimal. What overhead are we looking at in regards to code maintenance?

Reply:
we need to add an implementation of `AccountInfo` to `Account` under the `scroll` feature. I am a little worried of having this implementation and missing a shadowing of `AccountInfo` by `ScrollAccountInfo` combined with an `into()` which could be painful to debug.

Reply:
I pushed the revert of the deletion in a377aba, let me know if you want to keep it as is

Reply:
This looks fine to me